astro.config.mjs

```diff
@@ -1,24 +1,49 @@
 import { defineConfig } from "astro/config";
 import tailwind from "@astrojs/tailwind";
-import netlify from "@astrojs/netlify";
 import sanityImg from "@otterstack/sanity-img-astro/integration";
 import sanity from "@sanity/astro";
 import react from "@astrojs/react";
 import { loadEnv } from "vite";
-import liciousI18n from "@astrolicious/i18n";
 import { locales } from "./src/locales/consts";
+import paraglide from "@inlang/paraglide-astro";
+import sitemap from "@astrojs/sitemap";
+import netlify from "@astrojs/netlify";
+
 const {
   ASTRO_SITE_URL,
   SANITY_STUDIO_DATASET,
   SANITY_STUDIO_PROJECT_ID,
   SANITY_STUDIO_URL,
   SANITY_VISUAL_EDITING_ENABLED,
+  PUBLIC_ASTRO_BASE_PATH,
 } = loadEnv(process.env.NODE_ENV, process.cwd(), "");
 
+const basePath =
+  PUBLIC_ASTRO_BASE_PATH && PUBLIC_ASTRO_BASE_PATH !== ""
+    ? PUBLIC_ASTRO_BASE_PATH
+    : "/";
+
 // https://astro.build/config
 export default defineConfig({
   site: ASTRO_SITE_URL,
+  output: SANITY_VISUAL_EDITING_ENABLED === "true" ? "server" : "static",
+  adapter: netlify(),
+  base: basePath,
+  outDir: `./dist${PUBLIC_ASTRO_BASE_PATH}`,
+  i18n: {
+    defaultLocale: "en",
+    locales: locales,
+    routing: {
+      prefixDefaultLocale: true,
+    },
+  },
   integrations: [
+    sitemap(),
+    paraglide({
```

Comment:
I don't like that the i18n changed in this PR as well, since it doesn't have much to do with this ticket but ok. This will cause some problems with other PRs.. What's the reason for the new i18n?